### PR TITLE
Basic search page

### PR DESCRIPTION
**Pull request by finestructure**
Just the page for now, pagination is requiring bigger changes/discussion.

### CODE REVIEW

File: Sources/App/Views/Search/Search+View.swift [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        init(path: String, model: Model) {
+            self.model = model
+            super.init(path: path)
+        }
+
+        override func content() -> Node<HTML.BodyContext> {
+            .group(
+                .h2(.text("Results for \(model.query)")),
+                .ul(
+                    .class("list"),
```
Comment (finestructure) [inline Sources/App/Views/Search/Search+View.swift:19]: Don't think this class actually exists but `package_list` was rending the two column layout. Just a placeholder.

File: Sources/App/Core/SiteURL+siteMap.swift [hunk 2]
```diff
@@ -54,6 +54,8 @@ extension SiteURL {
                 return .hourly
             case .rssReleases:
                 return .hourly
+            case .search:
+                return .hourly
```
Comment (daveverwer) [inline Sources/App/Core/SiteURL+siteMap.swift:58]: It doesn't matter very much, but I'm not sure `hourly` is correct here. I was going to suggest `never` as the search page on its own is not something we'd want Google sending people to the search page directly, but the [docs](https://www.sitemaps.org/protocol.html) for `never` say:

> The value "never" should be used to describe archived URLs.

Which isn't true.

It's probably fine as `hourly`, but I thought I'd bring it up.

**Review by daveverwer — APPROVED** [review]
Tested locally. Only the comment about the sitemap really.

File: Sources/App/Core/SiteURL+siteMap.swift (hunk 2 above)
Comment (finestructure) [reply Sources/App/Core/SiteURL+siteMap.swift:58]: 👍  I don't really know what to pick here, I just filled in what I thought the change rate was going to be.

Comment (daveverwer) [reply Sources/App/Core/SiteURL+siteMap.swift:58]: It's just telling Google how often to come back and re-index that page., which should really be `never` as there will never be anything on that page for Google to index. It's not a huge deal

**Pull request merged**